<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="TO8K4MZL2-D" name="com.techouts.fanniemae.testcases.CreateContractMasterServicesAgreement" tests="1" failures="0" timestamp="2020-01-16T16:09:45 IST" time="91.052" errors="1">
  <testcase name="createContractMasterServicesAgreement" time="91.052" classname="com.techouts.fanniemae.testcases.CreateContractMasterServicesAgreement">
    <error type="com.techouts.fanniemae.exceptions.FannieMaeBusinessException" message="throwing exception to make sure capturing testcase result to testNG">
      <![CDATA[com.techouts.fanniemae.exceptions.FannieMaeBusinessException: throwing exception to make sure capturing testcase result to testNG
at com.fanniemae.automation.test.AbstractTest.performPostExceptionProcess(AbstractTest.java:244)
at com.techouts.fanniemae.testcases.CreateContractMasterServicesAgreement.createContractMasterServicesAgreement(CreateContractMasterServicesAgreement.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: com.techouts.fanniemae.exceptions.FannieMaeBusinessException: throwing exception to make sure capturing testcase result to testNG
at com.fanniemae.automation.test.AbstractTest.performPostExceptionProcess(AbstractTest.java:242)
... 29 more
Caused by: com.fanniemae.automation.util.exceptions.WebElementOperationException: Waited for page to load for 15 seconds.
at com.techouts.fanniemae.webelement.ops.WebElementOperations.throwWebElementOperationException(WebElementOperations.java:212)
at com.techouts.fanniemae.webelement.ops.WebElementOperations.waitForPageLoad(WebElementOperations.java:107)
at com.techouts.fanniemae.testcases.CreateContractMasterServicesAgreement.createContractMasterServicesAgreement(CreateContractMasterServicesAgreement.java:55)
... 28 more
Caused by: org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Some data haven't been filled correctly
  - Supplier : Mandatory
}
  (Session info: chrome=79.0.3945.117): Some data haven't been filled correctly
  - Supplier : Mandatory

Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'TO8K4MZL2-D', ip: '172.20.0.139', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_91'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 79.0.3945.117, chrome: {chromedriverVersion: 78.0.3904.70 (edb9c9f3de024..., userDataDir: C:\Users\srivani.a\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:50554}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 4bd9a0bd524d7d688181b1bf1237141a
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:120)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:489)
at com.techouts.fanniemae.webelement.ops.WebElementOperations.getExpectedCondition(WebElementOperations.java:135)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
at com.techouts.fanniemae.webelement.ops.WebElementOperations.waitForPageLoad(WebElementOperations.java:105)
... 29 more
]]>
    </error>
  </testcase> <!-- createContractMasterServicesAgreement -->
</testsuite> <!-- com.techouts.fanniemae.testcases.CreateContractMasterServicesAgreement -->
